clients/windows/src/input/mod.rs

```diff
@@ -5,5 +5,4 @@ pub mod message;
 pub mod mouse;
 pub mod pointer;
 
-pub use message::Message;
 pub use message::Point;
```

Comment:
Is this even worth keeping? I think the only place this is taken advantage of is here
https://github.com/lockbook/lockbook/blob/e92ed51631ff75ce67ea2527437c6f395dde0831/clients/windows/src/input/mouse.rs#L4

Reply:
Some clarification for my previous comment: When I said "this", I meant Point. Clippy was telling me to removed Message, but Point being used there allows shorthand use Point (Point instead of message::Point), However, it's only used in one place that doesn't actually make the line shorter. It's 
`use super::{message::MessageAppDep, Point}; `
instead of
`use super::message::{MessageAppDep, Point}; `

Hence, as far as I can tell, using Point is pointless

Reply:
I'm going to remove it unless told otherwise